numpy/core/setup_common.py

```diff
@@ -127,20 +127,12 @@ def check_api_version(apiversion, codegen_dir):
 # optional gcc compiler builtins and their call arguments and optional a
 # required header and definition name (HAVE_ prepended)
 # call arguments are required as the compiler will do strict signature checking
-OPTIONAL_INTRINSICS = [("__builtin_isnan", '5.'),
-                       ("__builtin_isinf", '5.'),
-                       ("__builtin_isfinite", '5.'),
-                       ("__builtin_bswap32", '5u'),
-                       ("__builtin_bswap64", '5u'),
-                       ("__builtin_expect", '5, 0'),
-                       ("__builtin_mul_overflow", '5, 5, (int*)5'),
-                       # MMX only needed for icc, but some clangs don't have it
+OPTIONAL_INTRINSICS = [# MMX only needed for icc, but some clangs don't have it
```

Comment:
This is a pretty massive deletion, what is the motivation?

Reply:
The current checks for whether or not to use these optional intrinsics is whether or not they compile. On this platform, they compile fine but do not work. So in this case, we just ignore them for this platform.